app/views/appointments/index.html.erb

```diff
@@ -1,2 +1,8 @@
 <h1>Appointments#index</h1>
 <p>Find me in app/views/appointments/index.html.erb</p>
+
+<%= button_to "Add Appointment",  new_appointment_path, :id => "open-contacts-dialog-btn",
```

Comment:
-Please check the syntax for ID, CLASS and méthod.
- Is it ChatGPT syntax? Can we use lewagon syntax?
- What is this ID name?
- What is this class name?